package.json

```diff
@@ -25,7 +25,7 @@
   "author": "Gustavo Santos (JBay Solutions) <gustavo.santos@jbaysolutions.com> (http://www.jbaysolutions.com)",
   "dependencies": {
     "element-resize-detector": "^1.1.10",
-    "interact.js": "^1.2.8"
+    "interact.js": "git://github.com/taye/interact.js.git#master"
```

Comment:
Or if you like you can put 
 `"interactjs": "^1.3.0-alpha.1"`
as a dependency but you have to require it from GridItem.vue like this:
   `var interact = require("interactjs/dist/interact.js");`
and it works like a charm :)
let me know if you need anorther PR